src/test/components/TrackScreenshots.test.js

```diff
@@ -247,22 +262,8 @@ function setup(
   const store = storeWithProfile(profile);
   const { getState, dispatch } = store;
   const flushRafCalls = mockRaf();
-  let leftOffset = LEFT;
-  let topOffset = TOP;
-  jest
-    .spyOn(HTMLElement.prototype, 'getBoundingClientRect')
-    .mockImplementation(() => {
-      const rect = getBoundingBox(TRACK_WIDTH, FULL_TRACK_SCREENSHOT_HEIGHT);
-      // Add some arbitrary X offset.
-      rect.left += leftOffset;
-      rect.right += leftOffset;
-      rect.x += leftOffset;
-      rect.y += topOffset;
-      rect.top += topOffset;
-      rect.bottom += topOffset;
```

Comment:
I'm not sure why there was an arbitrary X offset provided. Do you know why? I think we don't have this anymore.

Reply:
Initially, the `getBoundingBox` was only handling `width` and `height`, with the other values set as if the object was located at the top left of the screen. This offset is to position the object in another location. In the case of the screenshots, I believe this is so that we can test things like being close to the edges (top or right).

It's still implemented with the new tool, thanks to the parameters `offsetX/offsetY`.